core/stores/sqlx/utils.go

```diff
@@ -81,9 +81,9 @@ func format(query string, args ...interface{}) (string, error) {
 					return "", err
 				}
 
-				// index starts from 1 for pg
-				if index > argIndex {
-					argIndex = index
+				// index starts from 1 for pg or oracle
+				if index > 0 {
+					argIndex++
```

Comment:
this logic need to keep the original.

argIndex is not always go up in order. It might be [1, 3, 2 ...]